lms/djangoapps/instructor_task/tasks.py

```diff
@@ -290,3 +291,13 @@ def cohort_students(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('cohorted')
     task_fn = partial(cohort_students_and_upload, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+@task(base=BaseInstructorTask, routing_key=settings.GRADES_DOWNLOAD_ROUTING_KEY)  # pylint: disable=not-callable
```

Comment:
The routing key is used for uploading to S3, I believe. I think it makes sense for the ORA data downloads to live in the same place as all the other grade downloads.